web_page_archive.module

```diff
@@ -34,4 +35,7 @@ function web_page_archive_cron() {
   foreach ($config_entities as $id => $config_entity) {
     $cron_runner->run($config_entity);
   }
+
+  // Cleanup files.
+  CleanupController::processCleanup(50);
```

Comment:
Does the fixed parameter here feel weird?

Reply:
Yeah, I meant to create another issue to address this. I want to expose this as a configurable setting eventually but we need to build a place in the admin for global settings like this to live (i.e. Settings that should persist across all config entities). 

I suppose we could in the meantime replace this with the D8 equivalent of `variable_get`() and have that default to 50 or whatever. (Sorry. I'm writing this from my phone and didn't remember exact command for D8 off top of my head).

That would allow customization of this value via drush.

Reply:
Meh, you will address it when the time is right. Let's move forward ▶️